os/include/debug.h

```diff
@@ -1532,6 +1544,14 @@ int get_errno(void);
 /*        OS Function specific debug      */
 /******************************************/
 
+#ifdef CONFIG_SECURE_DEBUG_ASSERT
+#define is_enable_to_fault_output() get_security_level() == LOW_SECURITY_LEVEL
```

Comment:
1. For definition, let's use upper characters. `IS_ENABLE_....`.
2. Let's use brackets. `(get_security_.... == LOW....)`

Reply:
Thank you, Done